.gitlab-ci-check-docker-acceptance.yml

```diff
@@ -117,3 +117,41 @@ publish:acceptance:
         fi;
         sleep 1;
       done
+
+publish:acceptance:
+  stage: publish
+  except:
+    - /^saas-[a-zA-Z0-9.]+$/
+  image: golang:1.14-alpine3.11
+  dependencies:
+    - test:acceptance_tests
+  before_script:
+    - apk add --no-cache git
+    # Run go get out of the repo to not modify go.mod
+    - cd / && go get github.com/mattn/goveralls && cd -
+    # Coveralls env variables:
+    #  According to https://docs.coveralls.io/supported-ci-services
```

Comment:
hah, that is interesting. So at least we know now why it didn't work :roll_eyes: